README.md

```diff
@@ -36,22 +36,28 @@ NOTE: The default configuration will build 64-bit binaries for maximum security
 **IMPORTANT**: Currently, the `MAX_PATH` path length restriction (which is 260 characters by default) must be lifted in for our Python build scripts. This can be lifted in Windows 10 (v1607 or newer) with the official installer for Python 3.6 or newer (you will see a button at the end of installation to do this). See [Issue #345](https://github.com/Eloston/ungoogled-chromium/issues/345) for other methods for other Windows versions.
 
 1. Setup the following:
-
-    * 7-zip
-    * Python 3.6 - 3.9 or 3.10.2+ (for build and packaging scripts used below)
-        * At the end of the Python installer, click the button to lift the `MAX_PATH` length restriction.
-        * Check that your `PATH` does not contain the `python3` wrapper shipped by Windows, as it will only promt you to install Python from the Microsoft Store and exit. See [this question on stackoverflow.com](https://stackoverflow.com/questions/57485491/python-python3-executes-in-command-prompt-but-does-not-run-correctly)
+    * 7-Zip
+    * Python 3.6 - 3.9 or 3.10.2+ (for build and packaging scripts used below); Python 3.11 and above is not supported.
+    * If Python is not installed from the Microsoft Store:
```

Comment:
You can install Python in parallel with the version from the Microsoft Store.

Reply:
The point of the note is that the extra instructions (bar the removal of the `MAX_PATH` length restriction) don't need to be followed if the user is using the Microsoft Store version of Python, as half the instructions pertain to the installer and the other half prevent the Store version from launching.